src/orders/components/OrderListPage/OrderListPage.tsx

```diff
@@ -106,83 +117,114 @@ const OrderListPage: React.FC<OrderListPageProps> = ({
+                description: "tab name",
+              })}
+            />
+          </Box>
+
+          <Box display="flex" alignItems="center" gap={5}>
+            {!!onSettingsOpen && (
+              <CardMenu
+                className={classes.settings}
+                menuItems={[
+                  {
+                    label: intl.formatMessage({
+                      id: "vEwjub",
+                      defaultMessage: "Open in GraphiQL",
+                      description: "button",
+                    }),
+                    onSelect: openPlaygroundURL,
+                  },
+                  {
+                    label: intl.formatMessage({
+                      id: "WbV1Xm",
+                      defaultMessage: "Order Settings",
+                      description: "button",
+                    }),
+                    onSelect: onSettingsOpen,
+                  },
+                  ...extensionMenuItems,
+                ]}
+              />
             )}
-          />
-        )}
+            <ButtonWithSelect
+              disabled={limitsReached}
+              options={extensionCreateButtonItems}
+              data-test-id="create-order-button"
+              onClick={onAdd}
+            >
+              <FormattedMessage
+                id="LshEVn"
+                defaultMessage="Create order"
+                description="button"
+              />
+            </ButtonWithSelect>
+            {hasLimits(limits, "orders") && (
+              <LimitsInfo
+                text={intl.formatMessage(
+                  {
+                    id: "zyceue",
+                    defaultMessage: "{count}/{max} orders",
+                    description: "placed order counter",
+                  },
+                  {
+                    count: limits.currentUsage.orders,
+                    max: limits.allowedUsage.orders,
+                  },
+                )}
+              />
+            )}
+          </Box>
+        </Box>
       </TopNav>
       {limitsReached && <OrderLimitReached />}
       <Card>
-        <FilterBar
-          currentTab={currentTab}
+        <ListFilters
           initialSearch={initialSearch}
-          onAll={onAll}
           onFilterChange={onFilterChange}
           onSearchChange={onSearchChange}
-          onTabChange={onTabChange}
-          onTabDelete={onTabDelete}
-          onTabSave={onTabSave}
-          tabs={tabs}
-          allTabLabel={intl.formatMessage({
-            id: "WRkCFt",
-            defaultMessage: "All Orders",
-            description: "tab name",
-          })}
           filterStructure={filterStructure}
           searchPlaceholder={intl.formatMessage({
             id: "wTHjt3",
             defaultMessage: "Search Orders...",
           })}
         />
-        <OrderList {...listProps} />
+        <OrderListDatagrid
+          {...listProps}
+          onRowClick={id => {
+            navigate(orderUrl(id));
+          }}
```

Comment:
**thought:** Right now it is not possible to open datagrid items in new card using cmd+click or by right clicking the row. I remember this being highly requested feature, do you think it would be difficult to implement in the datagrid? Perhaps we could make it an issue and add it later on?

Reply:
It could be tricky imo, we  will have to develop our own context menu and this definitely not trivial